web/web/models/user.py

```diff
@@ -84,15 +84,15 @@ def does_user_role_exist(self, role_name):
         return False
 
     def __repr__(self):
-        return f'<User user_id={self.id} email_id={self.email}>'
+        return f'<User id={self.id} email_id={self.email}>'
 
 # Relationships
 Utility.users = relationship('User',
                             backref=db.backref('utility'))
 
 Address.user = relationship('User',
-                            backref=db.backref('address',
-                            uselist=False))
+                            backref=db.backref('address'),
+                            uselist=False)
```

Comment:
let's move these backref to their respective models as well

Reply:
The ones that are involved in the marshmallow schema as nested fields cannot be moved since I have to import AddressSchema. But I will go to each file and change the ones that can be moved to the other model. Thanks